src/dtx/dtx_srv.c

```diff
@@ -114,6 +114,13 @@ dtx_metrics_alloc(const char *path, int tgt_id)
 			D_WARN("Failed to create DTX RPC cnt metric for %s: "
 			       DF_RC"\n", dtx_opc_to_str(opc), DP_RC(rc));
 	}
+
+	rc = d_tm_add_metric(&metrics->dpm_total[DTX_PROTO_SRV_RPC_COUNT], D_TM_COUNTER,
+			     "total number of processed sync DTX_COMMIT", "ops",
+			     "%s/ops/sync_dtx_commit/tgt_%u", path, tgt_id);
```

Comment:
Looks good. So if i want to know how many piggy-backed commit notifications happened, i will be able to calculate it from this metrics + existing ones?

Reply:
This metrics stands for how many sync DTX commit (RPC) arrived from other DTX leaders.

Assume that:
1. The batched commit RPC (from others to current target) count is A.
2. The average bathed commit (from others to current target) degree is D.
3. The sync_dtx_commit (from others to current target) count is S.
4. The total DTX count on current target is T.
5. The count of DTX which leader resides on current target is L.

The DTX count that are committed via piggyback is equal to: T - L - (A * D + S). Unfortunately, we do not have metrics for "L", that contains both batched_commit and sync_commit from this target to others. If you want, I can add.

Reply:
maybe can add it to comment somewhere, I guess only you understand how to calculate it :)

Reply:
OK, I will add comment in subsequent DTX related patch.